server/server.go

```diff
@@ -510,17 +510,26 @@ func (s *Server) startMonitoring(secure bool) error {
 		hp           string
 		err          error
 		httpListener net.Listener
+		port   int
```

Comment:
What editor are you using? I would recommend that you include gofmt on save (VS Code with go plugin does that). 
That raises a question: we do `go fmt ./...` and it looks like Travis did not stop the build:
![image](https://user-images.githubusercontent.com/13736691/26935611-b376cbea-4c6c-11e7-945a-bebe068cd87a.png)
That's a bummer...